util/reggen/gen_cfg_html.py

```diff
@@ -113,33 +112,28 @@ def gen_cfg_html(cfgs: IpBlock, outfile: TextIO) -> None:
             act = ims.act
             width = str(ims.width) if ims.width is not None else "1"
             desc = ims.desc if ims.desc is not None else ""
-            genout(outfile,
-                   "    <tr>\n" +
-                   "      <td>" + name + "</td>\n" +
-                   "      <td>" + pkg_struct + "</td>\n" +
-                   "      <td>" + sig_type+ "</td>\n" +
-                   "      <td>" + act + "</td>\n" +
-                   "      <td>" + width + "</td>\n" +
-                   "      <td>" + desc + "</td>\n" +
-                   "    </tr>\n")
+            genout(
```

Comment:
nit: was this hit by an autoformatter? might be better to turn off formatting for these lines (and others in this file) as I think they were more readable before

Reply:
ah just saw the second commit message.

Reply:
Hum, `flake8` and `yapf` do not work well together.. I will skip yapf part.. Are `yapf` and `isort` optional now?

Reply:
I don't think they are enforced in CI at all, so no hehe.